resources/assets/js/forms.js

```diff
@@ -128,15 +148,23 @@ $(function () {
 /**
  * Show oauth buttons on welcome title click
  */
-$(function () {
-  $('#welcome-title').on('click', function () {
-    $('.btn-group.btn-group .btn.d-none').removeClass('d-none');
+ready(function () {
```

Comment:
arrow-function?

Reply:
Mention this one time per PR is sufficient :wink: